### PR TITLE
SG home

### PR DESCRIPTION
**Pull request by valentinludu**

### CODE REVIEW

File: apps/scoutgame/components/common/Header/components/HeaderMessage.tsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
 import 'server-only';
 
 import { Box, Typography } from '@mui/material';
+import { delay } from '@root/lib/utils/async';
 
-export function HeaderMessage() {
+export async function HeaderMessage() {
+  const data = await delay(3000);
```
Comment (mattcasey) [inline apps/scoutgame/components/common/Header/components/HeaderMessage.tsx:7]: is this for debugging or just to add a delay for the component to appear (which would be neat)?

File: apps/scoutgame/middleware.ts [hunk 2]
```diff
@@ -1,33 +1,26 @@
 import { getSession } from '@connect-shared/lib/session/getSession';
-import { isTruthy } from '@root/lib/utils/types';
 import type { NextRequest } from 'next/server';
 import { NextResponse } from 'next/server';
 
+const privateLinks = ['/profile', '/scout', '/notifications', '/u'];
```
Comment (mattcasey) [inline apps/scoutgame/middleware.ts:5]: should "/u" be private? does that mean viewing someone's profile requires login?

**Review by mattcasey — APPROVED** [review]

File: apps/scoutgame/components/common/Header/components/HeaderMessage.tsx (hunk 1 above)
Comment (valentinludu) [reply apps/scoutgame/components/common/Header/components/HeaderMessage.tsx:7]: It's for mocking a request and seeing a neat skeleton before the component appears. I have this everywhere. When the data will come from the backend this is the place to put them.

File: apps/scoutgame/middleware.ts (hunk 2 above)
Comment (valentinludu) [reply apps/scoutgame/middleware.ts:5]: Oh, no. /u is not private. I removed it.

**Pull request merged**